src/metabase/models/card.clj

```diff
@@ -105,16 +105,18 @@
 
 ;;; --------------------------------------------------- Revisions ----------------------------------------------------
 
+(def ^:private excluded-columns-for-card-revision
+  [:id :created_at :updated_at :entity_id :made_public_by_id :creator_id])
```

Comment:
Why is `:made_public_by_id` not in the revision, but `:public_uuid` is? The value should be kept in sync with `:public_uuid`, so either both of these columns need to be in the revision, or both of them need to be excluded.

Reply:
added 374884d4fd

I just noticed we're creating + adding public link using `/:card-id/public_link` and we didn't have any event for it, so it's probably not being recorded, will create an issue for it.

Reply:
created an issue here https://github.com/metabase/metabase/issues/31349